elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -42,6 +42,10 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
     new RestlasticSearchClient(new StaticEndpoint(new Endpoint(host, port)))
   }
 
+  override def refresh(): Unit = {
+    Await.result(restClient.refresh(index), 2.seconds)
```

Comment:
I wonder if we should name this something slightly different to differentiate what it's doing